src/css/birdbox-protocol-overrides.scss

```diff
@@ -0,0 +1,41 @@
+// custom CSS overrides for Birdbox, tweaking parts of Protocol
```

Comment:
@craigcook I'd welcome your take on whether this is OK to do, at least in the short-to-medium term, or if it's a hard no-no and we should make the fixes in core Protocol. 

(I think that the two newsletter-related CSS tweaks _will_ need to be moved into core Protocol when @alexgibson ports the Basket subscription JS from Bedrock into Protocol)

Reply:
It's fine when necessary (we do it often in Bedrock), but if these really are fixes that should also be done upstream let's file an issue in Protocol and try to make sure that happens soon.

Reply:
@craigcook Cool - and done: https://github.com/mozilla/protocol/issues/887